cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/UtilService.java

```diff
@@ -41,15 +43,17 @@
     public UtilService() {
     }
 
-    /** Get client's IP. */
+    /**
+     * Get client's IP.
```

Comment:
Ok. Now I understand what is doing this method. Can you explain why? what for? What functionality it's covered?